tests/get_account_test.go

```diff
@@ -0,0 +1,55 @@
+package tests
+
+import (
+	"github.com/Fantom-foundation/go-opera/ethapi"
+	"github.com/Fantom-foundation/go-opera/tests/contracts/transientstorage"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestGetAccount(t *testing.T) {
+	net, err := StartIntegrationTestNet(t.TempDir())
+	require.NoError(t, err, "failed to start the fake network")
+	defer net.Stop()
+
+	// Deploy the transient storage contract
+	_, deployReceipt, err := DeployContract(net, transientstorage.DeployTransientstorage)
```

Comment:
I think it would be more readable to deploy `Counter` - it would make it obvious that the simplest contract was deployed. Here I had to think for a moment why transient storage is needed

Reply:
Sure